docs/release.md

```diff
@@ -5,6 +5,27 @@
 - CAPRKE2 minor versions (v0.**2**.0 versus v0.**1**.0) are released every 3-4 months.
 - CAPRKE2 patch versions (v0.2.**2** versus v0.2.**1**) are released as often as weekly or once a month. 
 
+## Pre release operations
+
+Before publishing a new release, it must be added to `metadata.yaml`:
+```
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
+releaseSeries:
+  - major: 0
+    minor: 1
+    contract: v1beta1
+  - major: 0
+    minor: 2
+    contract: v1beta1
+  ...
+  - major: x
+    minor: x
+    contract: x
```

Comment:
Can this action be automated? So that a new TAG pushed will generate an update to the `metadata.yaml` and open a PR? This text only workflow is error prone.

Reply:
I thought the same but, since `metadata.yaml` must be updated before release is created, I think new tag pushed would not be a valid trigger, would it?

Reply:
Also https://cli.github.com/manual/gh_release_upload

Reply:
We need to update [e2e tests](https://github.com/rancher/cluster-api-provider-rke2/pull/372/files#diff-868dcd8976a79a1736e0bfe328a5100bd09c295509f0c3edd3f1d6baa72ff4aaR131-R132) and adjust e2e [configuration](https://github.com/rancher/cluster-api-provider-rke2/pull/372/files#diff-ab7361332a44af21b677fd39f4879f842f0b117aa60721a338967e49fe8894c4R91-R103) as well manually, so automating it at this point seems to be overkill to me.